deploy/helm/tests/logs-fargate-config-map_test.yaml

```diff
@@ -2,6 +2,8 @@
 suite: Test for logs-fargate-config-map
 templates:
   - logs-fargate-config-map.yaml
+chart:
+  version: 1.0.0
```

Comment:
Added explicit chart version so that the generated `sw.k8s.agent.manifest.version` in test snapshots does not change with each new release of the chart.